crates/extensions/c8y_mapper_ext/src/availability/builder.rs

```diff
@@ -32,7 +31,7 @@ pub struct AvailabilityBuilder {
 impl AvailabilityBuilder {
     pub fn new(
         config: AvailabilityConfig,
-        mqtt: &mut (impl MessageSource<MqttMessage, Vec<ChannelFilter>> + MessageSink<PublishMessage>),
+        mqtt: &mut (impl MessageSource<MqttMessage, Vec<ChannelFilter>> + MessageSink<MqttMessage>),
```

Comment:
That was subtle(accessing the MQTT actor indirectly via the mapper). Wondering if accepting `MessageSink` as a separate parameter and connecting directly to the MQTT actor would have made the connection flow easier follow.

Reply:
Yeah. My motivation was mostly to avoid changing the builders.